pubsub/schemas/create_proto_schema.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+// [START pubsub_create_proto_schema]
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func createProtoSchema(w io.Writer, projectID, schemaID, protoFile string) error {
+	// projectID := "my-project-id"
+	// schemaID := "my-schema"
+	// protoFile = "path/to/a/proto/schema/file(.proto)/formatted/in/protocol/buffers"
```

Comment:
Could we document this in a bit more detail, or link to some docs that's more specific about what's allowed?

e.g. what happens if I provide a proto file with multiple message definitions, or it contains rpc definitions, etc.

Reply:
I linked to the docs for now, but it doesn't provide a lot of context on how the proto definition works. I'll bring this up with the docs team at the next meetings.